specification/imagebuilder/resource-manager/Microsoft.VirtualMachineImages/stable/2022-02-14/imagebuilder.json

```diff
@@ -1226,6 +1226,7 @@
               "items": {
                 "$ref": "#/definitions/ImageTemplateInVMValidator"
               },
+              "x-ms-identifiers": [],
```

Comment:
Why is x-ms-identifier added only for "inVMValidations" array but not for other arrays - like "distribute"?
Regardless, I would much rather prefer removing it from here rather than adding everywhere else - to reduce churn.

Reply:
this change was made because it was causing the swagger lintdiff check to fail. the error is no longer showing as it's now been resolved but this is what it looked like:
![83293e8a-a951-410c-ae86-dc092d686a68](https://user-images.githubusercontent.com/90284189/167767398-3da2317e-dcae-4ca3-93d2-97bb01d41903.jpg)

Reply:
see this comment: https://github.com/Azure/azure-rest-api-specs/pull/18896#issuecomment-1116746656
I think the other places are not asked to be fixed since they were introduced before this rule was in place.
From the swagger model definition i don't think we have an identifier for those array objects. empty  x-ms-identifiers may be correct. @sumit-kalra thoughts?

Reply:
the extension being inProgress seems to be an issue with documentation according to honghr's comment. i think it might be better to keep it simple and only add this where the linting error indicates. from my experience these requirements are always changing and it might be something different during the next api release. it may be redundant to add this everywhere and then have to remove it in the future.